lib/streams.gi

```diff
@@ -1797,6 +1797,10 @@ InstallGlobalFunction( "UnInstallCharReadHookFunc",
 Unbind(OnCharReadHookActive);
 
 
+InstallValue(StandardInput, InputTextFile("*stdin*"));
+InstallValue(StandardOutput, OutputTextFile("*stdout*", true));
```

Comment:
Note that `colorprompt.g` also declares `STDOut` as ` BindGlobal("STDOut", OutputTextUser());`, which also binds it to `stdout`; so perhaps change that to use `StandardOutput`, too?

Also, `Exec` in `process.gi` uses stdin and stdout.